cmd/degitx-gitaly/main.go

```diff
@@ -0,0 +1,74 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"cqfn.org/degitx/config"
+	frontend "cqfn.org/degitx/front-end"
+	"cqfn.org/degitx/front-end/gitaly/server"
+	"cqfn.org/degitx/logging"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := cli.App{
+		Name:        "Gitaly-like front-end",
+		Usage:       "Manage degitx front-end",
+		Description: "DeGitX front-end CLI",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "config",
+				Aliases:  []string{"c"},
+				Usage:    "configuration file path",
+				Required: true,
+			},
+		},
+		Commands: []*cli.Command{
+			{
+				Name:    "run",
+				Aliases: []string{"r"},
+				Usage:   "run front-end",
+				Action:  cmdRun,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     "gitaly-host",
+						Usage:    "Gitaly gRPC API host and port",
+						Required: true,
+					},
+				},
+			},
+		},
+	}
+	if err := app.Run(os.Args); err != nil {
+		log.Fatalf("App failed: %s", err)
+	}
+}
+
+const pConfigUser = "${HOME}/.config/degitx/frontend/config.yml"
```

Comment:
@stepanvalyavskiy let's use `${HOME}/.config/degitx/frontend.yml"` for configuration to avoid unnecessary directory creating.